src/js/components/award/shared/activity/ContractGrantActivity.jsx

```diff
@@ -46,17 +62,92 @@ const ContractGrantActivity = ({
             window.removeEventListener('resize', handleWindowResize);
         };
     }, []);
+    /**
+     * X Translation
+     * We are positioning the potential award amount line tooltip centered.
+     * Therefore, we must halve the width to get the center of the graph and add 8
+     * (since 8 is half the pointer for the tooltip that is auto adjusted in the tooltip
+     * wrapper) and subtract half of the tooltip (since the tooltip position draws left to right)
+     * Y Translation
+     * We position with respect to the y position and add 8 for half the point and padding bottom.
+     */
+    const potentialAwardAmountLineTooltipData = (data) => ({
+        tooltipPosition: 'bottom',
+        styles: { // 8px is half the tooltip pointer, data.position is the y-position
+            transform: `translate(${((data.x2 / 2) + 8) - (defaultTooltipWidth / 2)}px,${data.position + 8 + defaultPadding.bottom}px)`,
+            position: 'absolute'
+        },
+        tooltipComponent: <RectanglePercentVizTooltip
+            title="Current Potential Award Amount"
+            amount={formatMoney(totalObligation)} />
+    });
+
+    const verticalLinesTooltipData = (data, text) => {
+        const date = moment(data.value);
+        const dateFormatted = date.format("MM/DD/YYYY");
+        const today = moment(Date.now());
+        const dateText = today.isAfter(date) ?
+            `${dateFormatted} (${convertDatesToRange(date, today)} ago)` :
+            `${dateFormatted} (${convertDatesToRange(today, date)} from today)`;
+        setThisLineOrTextIsHovered(text);
+        return {
+            styles: { // y position is in the middle of the line
+                position: 'absolute',
+                transform: `translate(${((data.position))}px,${(height / 2) - defaultPadding.bottom}px)`
```

Comment:
I see you're adjusting the position of the tooltip using the styles prop; which is fine, but the intent is really to have you use the `offsetAdjustment` prop for this purpose.